pkg/apis/scheduling/v1/helpers.go

```diff
@@ -0,0 +1,66 @@
+/*
```

Comment:
can we remove the pkg/apis/scheduling/helpers.go file?

Reply:
we're still using the helper funcs in the validation here: 

https://github.com/kubernetes/kubernetes/blob/ef830e2bcf7a23a0ae74e8d8b9e8811a83142f38/pkg/apis/scheduling/validation/validation.go#L37-L39

removing the file will make us do a explicit conversion to v1 api here which looks a bit odd to me. wdyt?

Reply:
no conversion needed, that method is checking name (string), value (int), and default (bool) matches